### PR TITLE
Additions to CTRL-2 for deployed mxcontrol cron logging

### PR DESCRIPTION
**Pull request by fcamilleri22**

### CODE REVIEW

File: src/handlers/mxapp.js [hunk 1]
```diff
@@ -13,7 +13,21 @@ const dynamoName = nconf.get("DYNAMODB:TABLE:MXCONTROL");
 const msgMXControlRoom =
     async message => hipchat.notify(message, {room: "MXControl"});
 
-const handleMXAppControlResponse = async ({source, message}) => {
+const handleMXAppResponse = async({source, message}) => {
+
+    if (source == "MXControl")
+        return await handleMXControlCronResponse(message);
+
+    //else
```
Comment (edobry) [inline src/handlers/mxapp.js]: bruh.

File: src/handlers/mxapp.js [hunk 2]
```diff
@@ -13,7 +13,21 @@ const dynamoName = nconf.get("DYNAMODB:TABLE:MXCONTROL");
...
+    return await handleMXControlStartupResponse(source);
+};
+
+const handleMXControlCronResponse = async(message) => {
```
Comment (edobry) [inline src/handlers/mxapp.js]: this `async/await` does nothing

File: src/handlers/mxapp.js [hunk 3]
```diff
@@ -13,7 +13,21 @@ const dynamoName = nconf.get("DYNAMODB:TABLE:MXCONTROL");
...
+    //this is easy. Pass the damn message.
```
Comment (edobry) [inline src/handlers/mxapp.js]: does this comment contribute

File: src/hooks/mxapp.js [hunk 4]
```diff
@@ -10,7 +10,8 @@ const parseMXAppTrigger = trigger =>
     }) => {
         const events = [];
 
-        if (message == "UP")
+        if ((message == "UP" && source != "MXControl")
```
Comment (edobry) [inline src/hooks/mxapp.js]: this condition can be simplified to `source == "MXControl" || message == "UP"`

**Review by edobry — APPROVED** [review]
its aight but the lil issues could use fixin

File: src/hooks/mxapp.js (hunk 4 above)
Comment (fcamilleri22) [reply src/hooks/mxapp.js]: should fail if `source == "MXControl" && message == "UP"`
 -- mxcontrol is an application and not an instance.

`source == "MXControl" || message == "UP"` allows this.

File: src/handlers/mxapp.js (hunk 3 above)
Comment (fcamilleri22) [reply src/handlers/mxapp.js]: will delete

File: src/handlers/mxapp.js (hunk 1 above)
Comment (fcamilleri22) [reply src/handlers/mxapp.js]: will delete

File: src/handlers/mxapp.js (hunk 2 above)
Comment (fcamilleri22) [reply src/handlers/mxapp.js]: Ordering of MXControl messages.

However, to reference a previous comment of yours - ordering is moot if you send everything as a single well formatted large message.

File: src/hooks/mxapp.js (hunk 4 above)
Comment (fcamilleri22) [reply src/hooks/mxapp.js]: fuck it this is dumb

**Pull request merged**